src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -243,6 +268,22 @@ private async Task<string> GetLanguage()
         return language;
     }
 
+    private string GetOverriddenLanguage()
+    {
+        // If the user has overridden the language from his profile by selecting a
+        // language directly in the form, then use the selected language.
+        if (_httpContextAccessor.HttpContext != null)
+        {
+            bool hasQueryLanguage = _httpContextAccessor.HttpContext.Request.Query.TryGetValue("lang", out StringValues queryLanguage);
```

Comment:
Could we add this to the controller in the `/next` endpoint, so that swagger will pick this up for documentation?

Reply:
I think I'm agreeing with @ivarne here, unless it would confuse api users as to where the language is used?

Reply:
Good point, both of you. It's an optional and very specific usage of the parameter and nothing to do with the process navigation, but still. I'll try to document it properly so we can have it in swagger without to much confusion.